src/map/battle.cpp

```diff
@@ -4071,6 +4071,14 @@ static void battle_calc_skill_base_damage(struct Damage* wd, struct block_list *
 						break;
 				}
 			}
+			if (skill_id == SN_SHARPSHOOTING || skill_id == MA_SHARPSHOOTING)
+				i &= ~(1); //Sharpshooting just ignores DEF but does not deal max damage
```

Comment:
Doesn't have to be done in this PR, but just a reminder for anyone. Might be beneficial to turn these flags into constants so we can easily read them without having to scroll around for hidden comments.

Reply:
Improved